.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,46 @@
+---
+name: ':beetle: Bug Report'
```

Comment:
I believe the emoji should be included directly as a character, otherwise, it will be rendered like this:
![image](https://user-images.githubusercontent.com/6367611/97146284-df476300-1767-11eb-9cd0-f1a9a7db1fd9.png)

See for instance Angular repository: https://raw.githubusercontent.com/angular/angular/90c0772743fda18e6b4bab6feaa731038b872cbc/.github/ISSUE_TEMPLATE/1-bug-report.md

Copying it from https://emojipedia.org/lady-beetle/ should work 🤔
```suggestion
name: '🐞 Bug Report'
```